### PR TITLE
OT90-635

### PR DESCRIPTION
**Pull request by andresmonsa**
Created the Loader component, It recibes {Time, height, width} for props. 
In case you don´t provide them, it has 3000, 100, 100 as a default value.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
       "react/jsx-props-no-spreading": "off",
       "no-console": "error",
       "semi": "off",
-      "no-param-reassign": "off"
+      "no-param-reassign": "off",
+      "linebreak-style" : 0
     }
 }
```
Comment (benjaminPla) [inline .eslintrc.json]: nice, can you comment what is happening here?

**Review by benjaminPla — APPROVED** [review]

File: src/Component/Loader.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import 'react-loader-spinner/dist/loader/css/react-spinner-loader.css'
+
+const Loader = ({ time = 3000, height = 100, width = 100 }) => (
+  <Spinner
+    type="Puff"
+    color="#00BFFF"
+    height={height}
+    width={width}
+    timeout={time}
```
Comment (dvelezroman) [inline src/Component/Loader.jsx]: what if a request takes more than three seconds to resolve from server ? how do we handle that ?
you might want better activate the spinner while some asyncronous callback is happening, or while loading data to the UI... I guess better apply those seconds for timeout when the async task has been completed... what do you think ?

File: .eslintrc.json (hunk 1 above)
Comment (andresmonsa) [reply .eslintrc.json]: > nice, can you comment what is happening here?

Not really bro. Is some stuff about eslint configuration.

**Review by verteramofederico — APPROVED** [review]

**Review by dvelezroman — APPROVED** [review]

**Review by Francisco-Merchan — APPROVED** [review]

File: src/Component/Loader.jsx (hunk 2 above)
Comment (andresmonsa) [reply src/Component/Loader.jsx]: I made some changes, I would like you to review them

**Review by dvelezroman — APPROVED** [review]

**Comment by andresmonsa** [comment]
Created the Loader component, It recibes {Visible, height, width} for props.
In case you don´t provide them, it has false, 500, 500 as a default value.

**Pull request merged**